addon.xml

```diff
@@ -1,103 +1,68 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="plugin.video.vrt.nu" name="VRT NU" version="2.2.3" provider-name="Martijn Moreel, dagwieers">
-    <requires>
-        <import addon="plugin.video.youtube" version="6.0.0" optional="true"/>
-        <!-- import addon="resource.images.studios.coloured" version="0.0.16" optional="true"/ -->
-        <import addon="resource.images.studios.white" version="0.0.21"/>
-        <import addon="script.module.beautifulsoup4" version="4.5.3"/>
-        <import addon="script.module.dateutil" version="2.6.0"/>
-        <!-- import addon="inputstream.adaptive" version="2.0.20" optional="true"/ -->
-        <import addon="script.module.inputstreamhelper" version="0.4.2"/>
-        <import addon="script.module.pysocks" version="1.6.8" optional="true"/>
-        <import addon="script.module.routing" version="0.2.3"/>
-        <import addon="xbmc.python" version="2.25.0"/>
-    </requires>
-    <extension point="xbmc.python.pluginsource" library="resources/lib/addon_plugin.py">
-        <provides>video</provides>
-    </extension>
-    <extension point="xbmc.service" library="resources/lib/addon_service.py"/>
-    <extension point="xbmc.python.module" library="resources/lib/"/>
-    <extension point="xbmc.addon.metadata">
-        <summary lang="en_GB">Watch videos from VRT NU</summary>
-        <description lang="en_GB">
-VRT NU is the video-on-demand platform of the Flemish public broadcaster (VRT).
+<requires>
+  <import addon="xbmc.python" version="2.25.0" />
+  <import addon="script.module.routing" version="0.2.3"/>
+  <import addon="script.module.inputstreamhelper" version="0.4.2"/>
```

Comment:
They used to be sorted alphabetically. I think it is a good practice to list things alphabetically when the order is of no importance.